helpers/version/action.yml

```
@@ -68,7 +68,7 @@ runs:
     - shell: bash
       run: |
         if [[ "${{ inputs.type }}" == "supervisor" ]]; then
-          sed -i "s/SUPERVISOR_VERSION .*/SUPERVISOR_VERSION = \"${{ steps.version.outputs.version }}\"/g" supervisor/const.py
+          sed -i "s/^SUPERVISOR_VERSION .*/SUPERVISOR_VERSION = \"${{ steps.version.outputs.version }}\"/g" supervisor/const.py
```

```suggestion
          sed -i "s/^SUPERVISOR_VERSION .*/SUPERVISOR_VERSION = \"${{ steps.version.outputs.version }}\"/" supervisor/const.py
```
Lets only do the first hit.

---

So I tested that and it still changed the value. I found [this post](https://stackoverflow.com/questions/9970124/sed-to-insert-on-first-match-only) and it seems like portable sed can't do first match only?

---

Thats lame..
How about `s/^SUPERVISOR_VERSION =.*` to make it more strict?